build/dependencies.yaml

```diff
@@ -150,6 +150,12 @@ dependencies:
       match: __default_debian_iptables_version=
     - path: test/utils/image/manifest.go
       match: configs\[DebianIptables\] = Config{list\.BuildImageRegistry, "debian-iptables", "[a-zA-Z]+\-v((([0-9]+)\.([0-9]+)\.([0-9]+)(?:-([0-9a-zA-Z-]+(?:\.[0-9a-zA-Z-]+)*))?)(?:\+([0-9a-zA-Z-]+(?:\.[0-9a-zA-Z-]+)*))?)"}
+  
+  - name: "k8s.gcr.io/setcap: dependents"
+    version: buster-v2.0.1
+    refPaths:
+    - path: build/common.sh
+      match: __default_setcap_version=
```

Comment:
Nit: Let's alpha-sort this, underneath the `pause: dependents` entries.

Reply:
@justaugustus -- Fixed, PTAL.